syntax = "proto3";

// TODO
// import "simple.proto";

enum RepeatedEnum {
  REP_UNKNOWN = 0;
  REP_ENUM_A = 1;
  REP_ENUM_B = 2;
  REP_ENUM_C = 3;
}

message RepeatedMessage {
  repeated int32 nums = 1;
  repeated string strs = 2;
  repeated RepeatedEnum enum_values = 3;
}